AmplifyPlugins/Logging/Sources/AWSCloudWatchLoggingPlugin/AWSCloudWatchLoggingSessionController.swift

```diff
@@ -164,7 +164,22 @@ final class AWSCloudWatchLoggingSessionController {
     }
     
     func flushLogs() async throws {
-        try await session?.logger.flushLogs()
```

Comment:
does this mean `flushLogs()` is no longer needed in `logger`?

Reply:
good catch, I've cleaned up and removed unused code.